source/include/VirtualInstrument.h

```diff
@@ -246,17 +248,17 @@ class ClumpParseState
     InstructionAllocator* _cia;
     
     Int32           _argCount;
-    void*           _argPointers[kMaxArguments];
-    TypeRef         _argTypes[kMaxArguments];
-    
-    Int32           _argPatches[kMaxArguments];     // Arguments that need patching
+    std::vector<void*> _argPointers;
```

Comment:
I have scheduled a meeting and discuss before using std::vector. I know there are a couple of  places vector is used already but before we proliferate the use of vector's we should talk. I think this commit is solving a separate defect by itself. Please this commit out of PR and submit it as a separate PR after we have a resolution on use of std::vector.